data/json/recipes/recipe_others.json

```diff
@@ -1887,7 +1940,9 @@
   "components": [
     [
       [ "string_36", 6 ],
-      [ "string_6", 36 ]
+      [ "string_6", 36 ],
+      [ "cordage_36", 6 ],
+      [ "cordage_6", 36 ]
```

Comment:
That would be one material I have seen some sources to claim has been used in ropes, and a rather smaller example is demonstrated here: https://www.youtube.com/watch?v=wAAw4AVgL04

It is also one of the most problematic things to make in a wilderness setting, at present.